samples/display/lvgl/prj.conf

```diff
@@ -6,6 +6,8 @@ CONFIG_DISPLAY_LOG_LEVEL_ERR=y
 CONFIG_LOG=y
 
 CONFIG_LVGL=y
+CONFIG_LVGL_HOR_RES=320
+CONFIG_LVGL_VER_RES=240
```

Comment:
Not sure if this is the resolution required for all demos, added them as defaults now are 640x480 (following lvgl latest defaults)

Reply:
As the demo is using static buffer allocation adding these settings look correct to me.

Reply:
This is one of the confusing configurations which is used only to configure the static buffer. I would like to keep the default configuration https://github.com/zephyrproject-rtos/zephyr/pull/26654/commits/9d9e442caa7175e2983fc3d373d66ed2d433d004#diff-9ddb187a0ec12686cef3456d1a500eecR13 as [320x240](https://github.com/zephyrproject-rtos/zephyr/blob/master/lib/gui/lvgl/Kconfig#L45) and do not add it to samples/display/lvgl/prj.conf

Reply:
I think that in this case it is better to not rely on defaults as sample should not break if they are changed.

Reply:
But it is You who did it be this patch, so please keep the defaults as 320x240 and remove it here.

Reply:
I think that the problem was on the sample, which relies on certain Kconfig defaults. I think that if a GUI requires a certain resolution it should specify it.

Reply:
That is only for the **buffer configuration**, display resolution is provided by the driver API (and the driver obtains is from the device tree, except native which has a Kconfig option for that), that is why I wrote it is a confusing configuration!

Reply:
I understand that it is for the buffer configuration, but still, won't it be dependent on the display used? Maybe it should be set in the board .conf files found in the sample folder so that sample can run on displays of different sizes?

Reply:
@jfischer-phytec-iot I have moved resolution and DPI settings on each board .conf file (hsdk board already does that). I think that it's the place where they should be. Just to take an example, `native_posix` stopped working if LVGL defaults were changed (it relied on defaults of 320x240).

Reply:
> That is only for the **buffer configuration**, display resolution is provided by the driver API (and the driver obtains it from the device tree, except native which has a Kconfig option for that), that is why I wrote it is a confusing configuration!

That about resoution is not quite true, I will check influence of LVGL_HOR_RES later.

Reply:
We could rename them `LVGL_MAX_HOR_RES ` and `LVGL_MAX_VER_RES ` (+ update Kconfig description) and only make them configurable/visible when `CONFIG_LVGL_BUFFER_ALLOC_STATIC` is set

Reply:
Yes.
I thought we already fixed it and always obtain resolution from the drivers. But it still not the case if LVGL_BUFFER_ALLOC_STATIC is enabled. I think we can rework it to obtain resolution from DT (need display bindings) with a fallback to Kconfig for native in the future.